app/docs/0.10.x/proxy.md

```diff
@@ -0,0 +1,759 @@
+- [Routing priorities][proxy-routing-priorities]
+- [Proxying behavior][proxy-proxying-behavior]
+    - [1. Load balancing][proxy-load-balancing]
+    - [2. Plugins execution][proxy-plugins-execution]
+    - [3. Proxying & upstream timeouts][proxy-proxying-upstream-timeouts]
+    - [4. Response][proxy-response]
+- [Configuring a fallback API][proxy-configuring-a-fallback-api]
+- [Configuring SSL for an API][proxy-configuring-ssl-for-an-api]
+    - [The `https_only` property][proxy-the-https-only-property]
+    - [The `http_if_terminated` property][proxy-the-http-if-terminated-property]
+- [Proxy WebSocket traffic][proxy-websocket]
+- [Conclusion][proxy-conclusion]
+
+[proxy-terminology]: #terminology
+[proxy-overview]: #overview
+[proxy-reminder]: #reminder-how-to-add-an-api-to-kong
+[proxy-routing-capabilities]: #routing-capabilities
+[proxy-request-host-header]: #request-host-header
+[proxy-using-wildcard-hostnames]: #using-wildcard-hostnames
+[proxy-preserve-host-property]: #the-preserve_host-property
+[proxy-request-uri]: #request-uri
+[proxy-strip-uri-property]: #the-strip_uri-property
+[proxy-request-http-method]: #request-http-method
+[proxy-routing-priorities]: #routing-priorities
+[proxy-proxying-behavior]: #proxying-behavior
+[proxy-load-balancing]: #1-load-balancing
+[proxy-plugins-execution]: #2-plugins-execution
+[proxy-proxying-upstream-timeouts]: #3-proxying-amp-upstream-timeouts
+[proxy-response]: #proxy-response
+[proxy-configuring-a-fallback-api]: #configuring-a-fallback-api
+[proxy-configuring-ssl-for-an-api]: #configuring-ssl-for-an-api
+[proxy-the-https-only-property]: #the-https_only-property
+[proxy-the-http-if-terminated-property]: #the-http_if_terminated-property
+[proxy-websocket]: #proxy-websocket-traffic
+[proxy-conclusion]: #conclusion
+
+---
+
+### Terminology
+
+In this document, we will refer to various Kong concepts. For the sake of 
+consistency, we will now define terms refering to such concepts:
+
+- `API`: This term refers to the API entity of Kong. You configure your APIs,
+that point to your own Upstream services through the Admin API.
+- `Plugin`: This refers to Kong "plugins", which are pieces of business logic
+that run in proxying lifecycle. Plugins can be configured globally (all 
+incoming traffic), or on a per-API basis through the Admin API.
+- `Client`: Refers to the - downstream - client making requests to Kong's proxy
+port.
+- `Upstream service`: Refers to your own API/service setting behind Kong,
+where requests are forwarded to.
+
+[Back to TOC](#table-of-contents)
+
+### Overview
+
+From a high level perspective, Kong will listen for HTTP traffic on its
+configured proxy port (`8000` by default), recognize which upstream service is
+being requested, run the configured plugins for that API, and forward the HTTP 
+request upstream to your own API or service. Kong can also run plugins once the 
+response from your upstream service is received, but that topic is not as
+relevant to this guide.
```

Comment:
> Kong can also run plugins once the response from your upstream service is received, but that topic is not as relevant to this guide.

Where do I find how to do this? Needs reference, otherwise remove.